receiver/cal_para_values.py

```diff
@@ -0,0 +1,29 @@
+temperature_lst = []
+chargeRate_lst = []
+soc_lst = []
+
+def get_para_list(para1, para2, para3):
+    temperature_lst.append(para1)
+    chargeRate_lst.append(para2)
+    soc_lst.append(para3)
+    return temperature_lst, chargeRate_lst, soc_lst
+
+
+def min_max_smavg(para1, para2, para3):

```

Comment:
It took me a while to realize that `min_max_smavg` is also about the global variables (`temperature_lst`, etc) themselves.
Consistency saves time for readers - either accept and return values in all functions (like `min_max_smavg`) or operate on the globals in all of them (like `get_para_list`)